sympy/sets/sets.py

```diff
@@ -555,25 +555,29 @@ def boundary(self):
     def is_open(self):
         """
         Property method to check whether a set is open.
+
         A set is open if and only if it has an empty intersection with its
-        boundary.
+        boundary. The openness of a subset of the reals is determined with
```

Comment:
I think that this could be more explicit. Maybe "a subset A of the reals is open if and only if each one of its points is contained in an open interval that is a subset of A". (This is equivalent to saying that A is the union of a family of open intervals.)

Reply:
Thanks for the suggestion, I've changed it.
I think I'll also rephrase the `is_closed` docstring to:
"A set is closed if its complement is an open set. In a metric space, in particular the reals, a set is closed if and only if it contains all of its limit points."

And I'll also change the added union test to test the actual results instead of only the fact that they're simplified.